Formula/opencascade.rb

```diff
@@ -0,0 +1,68 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  option "with-extras", "Install documentation (~17 MB), source files (~113 MB), samples and templates"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
+  depends_on "gl2ps"
+  depends_on "doxygen" if build.with? "extras"
+
+  def install
+    cmake_args = std_cmake_args
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    cmake_args << "-DCMAKE_FRAMEWORK_PATH:PATH=#{HOMEBREW_PREFIX}/Frameworks"
+    cmake_args << "-D3RDPARTY_DIR:PATH=#{HOMEBREW_PREFIX}"
+    cmake_args << "-DUSE_GL2PS:BOOL=ON"
+    cmake_args << "-DUSE_FREEIMAGE:BOOL=ON"
+    cmake_args << "-DUSE_TBB:BOOL=ON"
+    cmake_args << "-D3RDPARTY_TBB_DIR:PATH=#{HOMEBREW_PREFIX}"
+    cmake_args << "-D3RDPARTY_TCL_DIR:PATH=/usr"
+    cmake_args << "-D3RDPARTY_TCL_INCLUDE_DIR:PATH=#{sdk}/usr/include/"
+    cmake_args << "-D3RDPARTY_TK_INCLUDE_DIR:PATH=#{sdk}/usr/include/"
+
+    if build.with? "extras"
+      cmake_args << "-DINSTALL_SAMPLES=ON"
+      cmake_args << "-DINSTALL_DOC_Overview:BOOL=ON"
+    end
+
+    mkdir "build" do
+      system "cmake", "..", *cmake_args
+      system "make", "install"
+
+      if build.with? "extras"
+        prefix.install "../src"
+        prefix.install "../adm"
+        share.install_symlink prefix/"adm"
+      else
+        # Some apss expect resoures in legacy ${CASROOT}/src directory
+        cd prefix do
+          ln_s "share/opencascade/resources", "src"
+        end
+      end
+    end
+
+    # add symlinks to be able to compile against OpenCascade
+    loc = "#{prefix}/mac64/clang"
+    bin.install_symlink Dir["#{loc}/bin/*"]
+    lib.install_symlink Dir["#{loc}/lib/*"]
+  end
+
+  def caveats; <<-EOF.undent
```

Comment:
if we do keep the caveats, this needs to use the new style squiggly heredocs

Reply:
I think we should keep it.  I will change it to squiggly heredocs...